docs/05-gitlab/02-activation.mdx

```diff
@@ -3,6 +3,26 @@
 There are a few methods available, if you're using gitlab-ci, the easiest method in the current
 documentation is using gitlab-ci.
 
+:::caution Known issue with gitlab-ci activation and empty license file
```

Comment:
Copy paste: We will want to remove that once this is solved on the gitlab side, but this should help us a bit by pointing people in the right direction until this gets fixed. There has been a few people asking for this already on discord.